### PR TITLE
Kotlin conversion: RecentSearchDatabaseTable

### PR DESCRIPTION
**Pull request by sharvaniharan**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/search/RecentSearchDatabaseTable.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package org.wikipedia.search
+
+import android.content.ContentValues
+import android.database.Cursor
+import org.wikipedia.database.DatabaseTable
+import org.wikipedia.database.column.Column
+import org.wikipedia.database.contract.SearchHistoryContract
+
+class RecentSearchDatabaseTable : DatabaseTable<RecentSearch?>(SearchHistoryContract.TABLE, SearchHistoryContract.Query.URI) {
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/search/RecentSearchDatabaseTable.kt:9]: Can this be `DatabaseTable<RecentSearch>` (not nullable?)

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/search/RecentSearchDatabaseTable.java [hunk 2]
```diff
@@ -1,62 +0,0 @@
...
-
-import org.wikipedia.database.DatabaseTable;
-import org.wikipedia.database.column.Column;
-import org.wikipedia.database.contract.SearchHistoryContract;
-import org.wikipedia.database.contract.SearchHistoryContract.Col;
-
-import java.util.Date;
-
-public class RecentSearchDatabaseTable extends DatabaseTable<RecentSearch> {
-    private static final int DB_VER_INTRODUCED = 5;
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/search/RecentSearchDatabaseTable.java:16]: @sharvaniharan  This constant was `5`, but you changed it to `0` in the converted code. Why?

Comment (sharvaniharan) [reply app/src/main/java/org/wikipedia/search/RecentSearchDatabaseTable.java:16]: @dbrant did not change this... just did not verify after the auto convert 😱

Comment (dbrant) [reply app/src/main/java/org/wikipedia/search/RecentSearchDatabaseTable.java:16]: I went back to the code before this change and tried the auto-conversion to Kotlin.  It does not change the 5 to a 0.

**Pull request merged**